node/src/main/scala/coop/rchain/node/api/DeployGrpcService.scala

```diff
@@ -56,5 +55,8 @@ private[api] object DeployGrpcService {
         Observable
           .fromTask(defer(BlockAPI.showMainChain[F](request.depth)))
           .flatMap(Observable.fromIterable)
+
+      override def findBlockWithDeploy(request: FindDeployInBlockQuery): Task[BlockQueryResponse] =
+        defer(BlockAPI.findBlockWithDeploy[F](request.user, request.timestamp))
```

Comment:
@sebtomba Is this OK for casper to run its jobs on this `worker` scheduler? This is not a _short_ task (like dispatching) but CPU-bound.